module/index.js

```diff
@@ -3,8 +3,13 @@ const identity = require('1-liners/identity');
 
 export default (plugins) => {
   const transform = plugins.reduce(pipe, identity);
-  return (input) => {
-    const inputData = input.map((data) => ({data}));
-    return transform(inputData);
+  return (doxOutput) => {
+    const input = {
+      chunks: doxOutput.map((data) => ({data})),
+      version: '1',
```

Comment:
Yeah, I think I’ll do it so.

I thought the version should be very strict so that you can’t do `if (version > 1) {…}`. But on second thoughts:
* there’s nothing wrong with that
* `'2' > 1` is still `true` in JavaScript :)